orbs/pulumi.yml

```diff
@@ -208,3 +236,32 @@ examples:
             - pulumi/update:
                 stack: robot-co/website-prod
                 working_directory: ~/repo/src/website/
+
+  review_app_example:
```

Comment:
This example is still WIP but brought it from #15 to show how this orb will work.